README.md

```diff
@@ -86,101 +143,121 @@ The only required section is `site_info`
-1. The build pack sees the extension that we pushed and runs it.  The extension installs [wpcli](http://wp-cli.org/), and then reads your `setup.json` to install WordPress, and any plugins or themes you have defined. It then copies the rest of the files that we pushed and replaces the default Wordpress files with them.  In this case, it's just the `wp-config.php` file.
-1. At this point, the build pack is done and CF runs our droplet.
+We recommend using Cloud Foundry's "tasks" to run `wp-cli` commands. To do this, make sure to give the full path for both PHP and the `wp-cli.phar` file and specify the WordPress path relative to the `app` directory. Here's how you'd run `wp core version` on your cloud.gov container:
 
-### Changes
+```bash
+cf run-task APP_NAME "php/bin/php htdocs/wp-cli.phar core version --path='htdocs/'"
+```
 
-These changes were made to prepare Wordpress to run on CloudFoundry.
+That should print something like:
 
-1. Edit `wp-config.php`, configure to use CloudFoundry database.
+```
+Creating task for app APP_NAME in org ORG_NAME / space SPACE_NAME as USER_NAME...
+OK
 
-```diff
---- wp-config-sample.php	2013-10-24 18:58:23.000000000 -0400
-+++ wp-config.php	2014-03-05 15:44:23.000000000 -0500
-@@ -14,18 +14,22 @@
-  * @package WordPress
-  */
+Task has been submitted successfully for execution.
+task name:   98680974
+task id:     30
+```
 
-+// ** Read MySQL service properties from _ENV['VCAP_SERVICES']
-+$services = json_decode($_ENV['VCAP_SERVICES'], true);
-+$service = $services['cleardb'][0];  // pick the first MySQL service
-+
- // ** MySQL settings - You can get this info from your web host ** //
- /** The name of the database for WordPress */
--'DB_NAME', 'database_name_here');
-+'DB_NAME', $service['credentials']['dbname']);
+Run `cf logs APP_NAME --recent` to see the results and look for the `task name` to see the results. The task will create a container, run your command and then destroy the container after the task exits.
 
- /** MySQL database username */
--'DB_USER', 'username_here');
-+'DB_USER', $service['credentials']['username']);
+```
+2017-09-27T10:54:44.36-0600 [APP/TASK/98680974/0] OUT Creating container
+2017-09-27T10:54:44.81-0600 [APP/TASK/98680974/0] OUT Successfully created container
+2017-09-27T10:54:51.50-0600 [APP/TASK/98680974/0] OUT 4.8.2
+2017-09-27T10:54:51.52-0600 [APP/TASK/98680974/0] OUT Stopping instance 13abb9c4-23fe-4fc6-8b72-dc6676be26b8
+2017-09-27T10:54:51.51-0600 [APP/TASK/98680974/0] OUT Exit status 0
+2017-09-27T10:54:51.52-0600 [APP/TASK/98680974/0] OUT Destroying container
+2017-09-27T10:54:52.92-0600 [APP/TASK/98680974/0] OUT Successfully destroyed container
+```
 
- /** MySQL database password */
--'DB_PASSWORD', 'password_here');
-+'DB_PASSWORD', $service['credentials']['password']);
+Consider using continuous integration to run any tasks that should be run every time you `push` or `restage` your app.
```

Comment:
Append ", or that you want to run at regular time intervals."

Reply:
Should probably link [continuous integration] to https://cloud.gov/docs/apps/continuous-deployment/